### DISCUSSION TITLE
[Feat] When importing a dashboard include a version Note

### DISCUSSION THREAD

**Discussion by josepiratilla** [grafana/grafana#36995] [Dashboards, Panels & Visualizations]
When importing it creates a new version, but no note is included. I propose to include a predefined note (Like "Import"), or let the user to write it when importing.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
